src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  line-height: 14.6px;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  height: 60px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
+  height: 60px;
+}
+
+.logo {
+  margin-left: 50px;
+}
+
+.img {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  margin-left: 20px;
+  padding: 22.8px 0;
+  color: black;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  margin-left: 0;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  left: 397;
+  bottom: 0;
+  height: 4px;
+  width: 37px;
```

Comment:
This should not be fixed. And with `right` and `left` coordinates present, is useless.